paper-api/src/main/java/io/papermc/paper/plugin/loader/library/impl/MavenLibraryResolver.java

```diff
@@ -123,7 +123,7 @@ public void addRepository(final RemoteRepository remoteRepository) {
         if (MAVEN_CENTRAL_URLS.stream().anyMatch(remoteRepository.getUrl()::startsWith)) {
             LOGGER.warn(
                 "Use of Maven Central as a CDN is against the Maven Central Terms of Service. Use MavenLibraryResolver.MAVEN_CENTRAL_DEFAULT_MIRROR instead.",
-                new RuntimeException("Plugin used Maven Central for library resolution")
+                new RuntimeException("Plugin used Maven Central for library resolution of [%s]".formatted(remoteRepository.toString()))
```

Comment:
The way this is currently worded makes it seem like the library being resolved is the repository url, something like (url: %s) could be clearer.

Reply:
You mean the whole Throw message?

Reply:
I mean only the part being added here
```suggestion
                new RuntimeException("Plugin used Maven Central for library resolution (url: %s)".formatted(remoteRepository.toString()))
```

Reply:
what about?
```suggestion
                new RuntimeException("Plugin used Maven Central for library resolution (%s)".formatted(remoteRepository.toString()))
```

because the toString give a little more than just the url.

Reply:
I assumed it'd only show url so that's fine then